### PR TITLE
Registration, log in, log out done. Shown email if logged in. Shown "log out" if logged in. Shown "log in" if logged out. Password encrypted

### PR DESCRIPTION
**Pull request by Sheff3rd**
Screens updated!
Registration without reload:
screenshot from 2016-11-18 01-38-58
Signed up:
screenshot from 2016-11-18 01-39-07
Authentication without reload:
screenshot from 2016-11-18 01-39-19
Logged in:
screenshot from 2016-11-18 01-39-32
After js page reload:
screenshot from 2016-11-18 01-39-42
Logged out:
screenshot from 2016-11-18 01-39-50

### CODE REVIEW

**Comment by galulex** [comment]
:+1:

**Comment by galulex** [comment]
The main issue is that I can see page reloads here. Could you do everything remote please?

**Comment by Sheff3rd** [comment]
Welp I couldn't do dynamic 'login' and 'logout' panels, so that they switch themselves when i log in or log out. So I've done 'location.reload()' function in my .js.erb files for now
I'll look into it some more

File: app/views/tasks/index.html.erb [hunk 1]
```diff
@@ -3,6 +3,17 @@
 <%= link_to 'Done', filtered_tasks_path(:done) %>
 |
 <%= link_to 'Open', filtered_tasks_path(:open) %>
+<br />
+<div class = 'panel'>
+<% if session[:user_id] %>
```
Comment (galulex) [inline app/views/tasks/index.html.erb]: `if current_user`

File: app/views/tasks/index.html.erb [hunk 2]
```diff
@@ -3,6 +3,17 @@
...
+  <%= link_to 'Log Out', log_out_path, remote: true %><br />
+  <%= current_user.email %>
+<% else %>
+  <%= link_to 'Sign Up', sign_up_path, remote: true%>
+  |
+  <%= link_to 'Log In', log_in_path, remote: true %><br />
+<% end %>
+</div>
```
Comment (galulex) [inline app/views/tasks/index.html.erb]: extract it to partial `layouts/_panel`

File: app/views/sessions/create.js.erb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+$('#new_session').remove();
+$('.panel').show();
```
Comment (galulex) [inline app/views/sessions/create.js.erb]: `$('.panel').replaceWith("<%= j(render partial: 'layouts/panel') %>")`

**Comment by Sheff3rd** [comment]
Without page reload:

screenshot from 2016-11-18 16-45-50
screenshot from 2016-11-18 16-46-31
screenshot from 2016-11-18 16-46-40
screenshot from 2016-11-18 16-46-52
screenshot from 2016-11-18 16-47-06

**Pull request closed without merging**